<launch>

    <!-- Robot Agruments -->
    <arg name="ROBOT_IP"       default="192.168.2.40"/>
    <arg name="enable_gripper" default="True"/>
    <arg name="asynchronous"   default="False"/>
    <arg name="limit_acc"      default="True"/>
    <arg name="ft_sensor"      default="True"/>

    <!-- RTDE - Position Controller -->
    <node pkg="ur_rtde_controller" type="rtde_controller" name="ur_rtde_controller" output="screen">
        <param name="ROBOT_IP"       value="$(arg ROBOT_IP)"/>
        <param name="enable_gripper" value="$(arg enable_gripper)"/>
        <param name="asynchronous"   value="$(arg asynchronous)"/>
        <param name="limit_acc"      value="$(arg limit_acc)"/>
        <param name="ft_sensor"      value="$(arg ft_sensor)"/>
    </node>

</launch>
